lib/ansible/modules/test/units/modules/cloud/google/test_gce_tag.py

```diff
@@ -0,0 +1,66 @@
+import unittest
```

Comment:
You need to add `#!/usr/bin/python` to the first line here, and that should fix up your tests on shippable.